lib/galaxy/managers/histories.py

```diff
@@ -207,6 +208,12 @@ def p_tag_filter(term_text: str, quoted: bool):
                 self.model_class.deleted == (true() if show_deleted else false())
             )
 
+        # By default, only return non-archived histories when we are showing the current user's histories
```

Comment:
```suggestion
        # By default, return only non-archived histories when we are showing the current user's histories
```
Also, tiny, tiny change. But now it reads to me as "select ONLY non-archived histories when condition is true" vs. "only do this operation when condition is true". I think it's clearer this way? (unless I'm misunderstanding the logic)